core/src/components/checkbox/checkbox.stories.js

```diff
@@ -0,0 +1,53 @@
+export default {
+  title: "Components/Checkbox (WIP)",
+};
+
+// TODO: Add Selected and SelectedAndDisabled stories after icons are available
+
+export const Default = () => `
+  <label class="fudis-checkbox">
+    <input type="checkbox" class="fudis-checkbox__input"></input>
+    <div class="fudis-checkbox__content">
+       <div class="fudis-checkbox__content-wrapper">
+        <span class="fudis-checkbox__content__box"></span>
+       </div>
+       <div class="fudis-checkbox__content__label">Option</div>
```

Comment:
In Fudis we have all of these "fudis-checkbox__content__label" as span elements.

Reply:
Good spotting, changed all these to `span` elements.